MidCamp/midcamp
# Basic theming

### Related
This PR will be enhanced with some styling updates to Hatter, specifically:
* ~~https://github.com/MidCamp/Hatter/pull/5~~ **merged**
* ~~https://github.com/MidCamp/Hatter/pull/4~~ **merged**
* ~~https://github.com/MidCamp/Hatter/pull/6~~ **merged**

#### Tickets
* https://github.com/orgs/MidCamp/projects/1#card-4865281
* https://github.com/orgs/MidCamp/projects/1#card-4865312
* https://github.com/orgs/MidCamp/projects/1#card-4865316

#### Testing instructions
* `drush cim`
* `drush cr`
* Add some main nav links
* Create a taxonomy term for an event
* Create a sponsor (or three) and reference the event
* Create a venue and reference the event
* Create an article (or multiple) that reference the event
* Set the event as the hompage
* Navigate to homepage
* Observe that you see the main nav links below the banner image
* Observe that you see the footer links in the footer 
* Observe that you see the copyright in the footer
* Observe that you see the Sponsor logos
* Observe that the sponsor logos are linked to the node page
* Observe that the venue information appears in the map overlay
* Observe that you see the News teasers
* Navigate to a page other than the homepage
* Observe that you see the banner, menu, and footer
* Observe that they appear like the styleguide (https://midcamp.github.io/Hatter/home/) 
* Observe that the main content area is wrapped in a `.container` 

#### Remaining tasks
* [x] Fix MapBox
* [ ] Update the copyright information with the correct content

#### Content updates (once deployed)
* [ ] Add real menu links to the main nav
* [ ] Add real menu links to the footer nav
* [ ] Update sponsor links on sponsor view
* [ ] Create footer block to match prototype

#### New tickets/Out of scope
* [x] Figure out how news should be displayed on the homepage
* [x] Figure out how events should be displayed on the homepage
* [ ] Add "Featured speakers" block once attendees are working correctly
